binding/Binding/Skottie/InvalidationController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SkiaSharp.Skottie
+{
+	public unsafe class InvalidationController : SKObject
```

Comment:
If objects are only ever created by users, we can also add the `ISKSkipObjectRegistration` interface to speed up and also reduce locks since we will not need to keep a reference to the handle.